src/common/snippets/src/pass/mha_tokenization.cpp

```diff
@@ -472,9 +472,24 @@ ov::snippets::pass::TokenizeMHASnippets::TokenizeMHASnippets(const SnippetsToken
 
         // TODO [75567]: move this plugin-specific constraint to the plugin callback
         const auto last_node = ordered_ops.back();
-        if (potential_body_params_count + last_node->get_output_size() + hidden_virtual_ports_count + uniqie_buffer_reg_group_count > 11) {
+        const auto io_count =  potential_body_params_count + last_node->get_output_size() + hidden_virtual_ports_count;
+        const auto data_count = io_count + uniqie_buffer_reg_group_count;
+        auto available_regs = config.get_data_ptr_gpr_count();
+        // [150148, 150149] Currently Snippets don't have mechanism of spilling registers on stack.
+        //                  Due to this limitation we have to not tokenize some subgraphs if we need more registers than we have on target machine.
```

Comment:
```suggestion
        //                  Due to this limitation we have to skip tokenization of some subgraphs if we need more registers than we have on the target machine.
```

Reply:
If you let me, I will fix it in the separate commit if there are other concerns which should be fixed 😃 
If there are no other major comments, I will fix in the follow-up PR

Reply:
Ok, this is very minor for sure.